quelpa.el

```diff
@@ -1759,11 +1767,12 @@ So here we replace that with `insert-file-contents' for non-tar files."
         (package-install-file file))
     (package-install-file file)))
 
-(defun quelpa-package-install (arg)
+(defun quelpa-package-install (arg &rest plist)
   "Build and install package from ARG (a recipe or package name).
+PLIST is a plist that may modify the build and/or fetch process.
 If the package has dependencies recursively call this function to install them."
   (let* ((rcp (quelpa-arg-rcp arg))
-         (file (and rcp (quelpa-build rcp))))
+         (file (and rcp (quelpa-build (append rcp plist)))))
```

Comment:
Nitpick: Please use `when` instead of `and` to make the conditional more clear.

Reply:
Done